tiles/src/main/java/com/protomaps/basemap/layers/Boundaries.java

```diff
@@ -199,6 +199,13 @@ public void processFeature(SourceFeature sf, FeatureCollector features) {
             // Natural Earth is used for low zooms (for compilation and tile size reasons)
             themeMinZoom = 6;
           }
+          // used in Colombia, Brazil, Kenya (historical)
```

Comment:
Beware: In Brazil there are several "statistical" groupings of actual administrative regions. Some of these are in conventional usage, some are not. I'm also wary of adding historical boundary lines. However, since the map looks broken without this change we should make it... but worth some more research...